hazelcast/src/main/resources/hazelcast-config-5.0.json

```diff
@@ -3281,6 +3281,10 @@
           "type": "boolean",
           "default": false
         },
+        "auto-remove-stale-data": {
+          "type": "boolean",
+          "default": false
```

Comment:
The default value is `true`. Also, you could copy the [documentation from the XSD](https://github.com/hazelcast/hazelcast/blob/462045ca037af733bded32375ea4d7e96b52628a/hazelcast/src/main/resources/hazelcast-config-5.1.xsd#L3927-L3935) into the json schema `"description"` field, but now I see that the documentation has to be updated to reflect changes we made in 5.0 persistence, so feel free to create a separate issue for that.

Reply:
Thanks for the catch and comment :) I [opened](https://github.com/hazelcast/hazelcast/issues/19708) an issue regarding (config) documentation update on persistence